### ISSUE TITLE
Error / incorrect parsing of ISBN wikitext template. 

### ISSUE THREAD

**Issue by MarketingPip** [spencermountain/wtf_wikipedia#565]
This is pulled from a references() object. 

```json
"isbn":"0-7735-0851-1}}; {{cite book |last= Walker|first= James W. St. G.|author-link=James W. St. G. Walker|date= 1997|title= 'Race,' rights and the law in the Supreme Court of Canada|location= Waterloo|publisher= Wilfrid Laurier University Press|pages= 182–245|isbn= 0-88920-289-3|undefined"
```

**Comment by spencermountain** [comment]
hey Jared, what page is this from?
I tried [James_W._St._G._Walker](https://en.m.wikipedia.org/wiki/James_W._St._G._Walker#/editor/0) and don't see this issue - can you help me reproduce it?
cheers

**Comment by MarketingPip** [comment]
My apologies - didn't mean to not throw something down for you. (Bad practice on my part - but was a 3AM find & figured I'd log it before bed and forgot).

I'll see if I can find you something tonight. Cheers 🍻

**Comment by spencermountain** [comment]
think this should be fixed in 10.2.1, cheers
